src/main/java/io/supertokens/inmemorydb/queries/SessionQueries.java

```diff
@@ -68,7 +72,8 @@ public static void createNewSession(Start start, String sessionHandle, String us
             pst.setString(4, userDataInDatabase.toString());
             pst.setLong(5, expiry);
             pst.setString(6, userDataInJWT.toString());
-            pst.setLong(7, createdAtTime);
+            pst.setString(7, grantPayload == null ? null : grantPayload.toString());
```

Comment:
Should we store null in the db? I'm not sure. Do we allow `userDataInJWT` or `userDataInDatabase` to be `null`? If not, then maybe we should store an empty object?

Reply:
The thinking was that this would be null for V2 tokens, this way we have a way to differentiate them when fetching and we don't need to add a default value.
`jwt_user_payload` and `session_data` both allow nulls.